### PR TITLE
[Fix Issue #4] Add "How to contribute" section to README

### PR DESCRIPTION
**Pull request by Gansito144**
* Including some links

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -19,3 +19,18 @@ Clone this repo manually or use `brunch new dir -s brunch/with-react`
     * `public/` dir is fully auto-generated and served by HTTP server.  Write your code in `app/` dir.
     * Place static files you want to be copied from `app/assets/` to `public/`.
     * [Brunch site](http://brunch.io), [Getting started guide](https://github.com/brunch/brunch-guide#readme)
+
+## How to contribute
+Contribution is very welcome, you can review [current issues](https://github.com/algoristas/dashboard/issues) or request a new one by creating a new issue.
```
Comment (rendon) [inline README.md]: Pluralization: Contribution*s* are

File: README.md [hunk 2]
```diff
@@ -19,3 +19,18 @@ Clone this repo manually or use `brunch new dir -s brunch/with-react`
...
     * [Brunch site](http://brunch.io), [Getting started guide](https://github.com/brunch/brunch-guide#readme)
+
+## How to contribute
+Contribution is very welcome, you can review [current issues](https://github.com/algoristas/dashboard/issues) or request a new one by creating a new issue. 
+
+### Pull Request process
+Once you have the issue/feature to work on, follow these steps to submit your changes and  make a pull request of them.
+
+* Inside repository create a new branch `git checkout -b <your_branch>`
+* Made your changes and add them `git add <list of your files>`
```
Comment (rendon) [inline README.md]: Make?

File: README.md [hunk 3]
```diff
@@ -19,3 +19,18 @@ Clone this repo manually or use `brunch new dir -s brunch/with-react`
...
+* Create a commit with an appropiate message `git commit` [guidelines](https://chris.beams.io/posts/git-commit/)
```
Comment (rendon) [inline README.md]: I think most people know they have to commit their changes, I would just say something like: _Try to follow the guidelines at <the link> for your commits_.

**Review by rendon — APPROVED** [review]
Thanks for adding this info, just a couple suggestions.

File: README.md [hunk 4]
```diff
@@ -21,15 +21,14 @@ Clone this repo manually or use `brunch new dir -s brunch/with-react`
     * [Brunch site](http://brunch.io), [Getting started guide](https://github.com/brunch/brunch-guide#readme)
 
 ## How to contribute
-Contribution is very welcome, you can review [current issues](https://github.com/algoristas/dashboard/issues) or request a new one by creating a new issue. 
+Contribution are welcome, you can review [current issues](https://github.com/algoristas/dashboard/issues) or create a new one.
```
Comment (rendon) [inline README.md]: You missed the *s* in Contribution :)

File: README.md [hunk 5]
```diff
@@ -21,15 +21,14 @@ Clone this repo manually or use `brunch new dir -s brunch/with-react`
     * [Brunch site](http://brunch.io), [Getting started guide](https://github.com/brunch/brunch-guide#readme)
 
 ## How to contribute
-Contribution is very welcome, you can review [current issues](https://github.com/algoristas/dashboard/issues) or request a new one by creating a new issue. 
+Contribution are welcome, you can review [current issues](https://github.com/algoristas/dashboard/issues) or create a new one. 
 
 ### Pull Request process
-Once you have the issue/feature to work on, follow these steps to submit your changes and  make a pull request of them.
+Once you selected issue to work on, follow these steps to submit your changesi, then create a pull request.
```
Comment (rendon) [inline README.md]: changesi?

**Review by rendon — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (Gansito144) [reply README.md]: Damn it!

File: README.md (hunk 5 above)
Comment (Gansito144) [reply README.md]: I blame vim for this :p

**Review by rendon — APPROVED** [review]
Do it!

**Pull request merged**